meetuppizza/views.py

```diff
@@ -1,8 +1,52 @@
 from django.http import HttpResponse
 from django.template import loader
 from django.shortcuts import render
+from django.shortcuts import redirect
+from django.contrib.auth.models import User
+from django.contrib.auth import authenticate, login, logout
+from meetuppizza.forms import RegistrationForm
+import pdb
 
 def index(request):
-  # template_name = "template/index.html"
-  # template = loader.get_template('templates/index.html')
-  return render(request, 'index.html')
\ No newline at end of file
+  return render(request, 'index.html')
+
+def sign_up(request):
+  if request.method == 'GET':
+    args = {}
```

Comment:
The thing called `args` here is usually called the template [context](https://docs.djangoproject.com/es/1.9/ref/templates/api/#django.template.Context).